src/components/OrganizationCard/OrganizationCard.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import styles from './OrganizationCard.module.css';
+
+interface Details {
+  key: any;
+  image: string;
+  id: string;
+  name: string;
+  lastName: string;
+  firstName: string;
+}
```

Comment:
Please use a descriptive name for your interfaces. `Details` does not indicate the purpose of this interface.

See the suggestion below
```suggestion
interface OrganizationCardProps {
  key: any;
  image: string;
  id: string;
  name: string;
  lastName: string;
  firstName: string;
}
```